kernel/solo5.h

```diff
@@ -136,8 +165,13 @@ uint64_t solo5_clock_monotonic(void);
 uint64_t solo5_clock_wall(void);
```

Comment:
`clock_gettime()` can fail due to (on Linux):
- `EFAULT` tp points outside the accessible address space.
- `EINVAL` The clk_id specified is not supported on this system.

We should never get `EFAULT` (since we've already verified the buffer passed by the guest). If we do then we want to terminate the guest with extreme prejudice anyway. We will never get `EINVAL` since `CLOCK_MONOTONIC` is supported on any sane system.